apps/web/src/lib/thread.ts

```diff
@@ -1,8 +1,21 @@
 import { Thread } from "@langchain/langgraph-sdk";
 import { getMessageContentString } from "@open-swe/shared/messages";
-import { GraphState } from "@open-swe/shared/open-swe/types";
+import { GraphState, TaskPlan } from "@open-swe/shared/open-swe/types";
 import { getActivePlanItems } from "@open-swe/shared/open-swe/tasks";
 
+export function computeThreadTitle(
+  taskPlan: TaskPlan | undefined,
+  fallbackTitle: string,
+): string {
+  if (taskPlan?.tasks && taskPlan.tasks.length > 0) {
+    const firstTaskTitle = taskPlan.tasks[0]?.title;
```

Comment:
just need to wrap in a try/catch since `getActiveTask` will error if no tasks exist
```suggestion
    const firstTaskTitle = getActiveTask(taskPlan).title;
```

Reply:
I tried this however I believe we want the `const firstTaskTitle = taskPlan.tasks[0]?.title;` because getActivetask does not always grab the plan title (plan title is first task only)